client/verta/verta/_internal_utils/_artifact_utils.py

```diff
@@ -200,6 +202,16 @@ def ensure_bytestream(obj):
             bytestream.seek(0)
             return bytestream, "cloudpickle"
 
+        torch = maybe_dependency("torch")
+        if torch is not None:
+            try:
+                torch.save(obj, bytestream)
+            except:  # not something torch can serialize
+                pass
+            else:
+                bytestream.seek(0)
+                return bytestream, "torch"
+
```

Comment:
Is this ever going to be reached? From a cursory looks, it seems like the cloudpickling will be attempted first, correct? Which should succeed for torch model (that's how our existing passed)? Could you verify that this will be called during log_model (e.g by adding a print statement here, then try to run some of the tests I mentioned above).